core/templates/dev/head/pages/exploration_editor/exploration_title_editor_directive.html

```diff
@@ -0,0 +1,22 @@
+<p ng-class="{'has-error': !explorationTitleService.displayed.length > 0}">
+  <label for="explorationTitle" ng-style="<[::formStyle]>"><[::labelText]></label>
+  <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" 
+         ng-model="explorationTitleService.displayed" ng-blur="onInputFieldBlur()" 
+         placeholder="Choose a title for your exploration." focus-on="<[::focusLabel]>" 
+         maxlength="40" ng-trim="false">
+  <span class="help-block" style="font-size: smaller">
+    <em>Please use at most 40 characters, so that this fits in the summary card.</em>
+    <span ng-if="explorationTitleService.displayed.length > 0">
+      <em>Characters used: <[explorationTitleService.displayed.length]>/40</em>
+    </span>
+  </span>
+</p>
+<style>
+@media (min-width: 768px) {
```

Comment:
Indent this and the following lines by 2. (Again, follow existing codebase style please.)